_config.yaml

```diff
@@ -36,8 +36,8 @@ header_pages:
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
   - workshop/workshop-materials.md
-  - workshop/resources-for-consultation-sessions.md
-  - working-with-your-data/working-with-your-own-data.md
+  - workshop/workshop-resources.md
+  - workshop/working-with-your-own-data.md
```

Comment:
Reviewers: these are the files which will appear in the navbar of the website. Note that later, `workshop-structure.md` will get replaced with something a bit more dynamic as described in #115